items.json

```diff
@@ -3176,10 +3176,10 @@
           "MeleeWeapon",

```

Comment:
Not usable with onehand yet it have a onehand mode. This lowers the budget for the weapon. Remove onehanded mode. To do that remove each pieces for the onehanded weapondescriptions in weapondescriptions.xml

Reply:
discussed in discord/pretty sure this is just a tag on the 2h mode